Formula/go.rb

```diff
@@ -4,9 +4,9 @@ class Go < Formula
   license "BSD-3-Clause"
 
   stable do
-    url "https://dl.google.com/go/go1.14.6.src.tar.gz"
-    mirror "https://fossies.org/linux/misc/go1.14.6.src.tar.gz"
-    sha256 "73fc9d781815d411928eccb92bf20d5b4264797be69410eac854babe44c94c09"
+    url "https://dl.google.com/go/go1.15.src.tar.gz"
```

Comment:
```suggestion
    url "https://golang.org/dl/go1.15.src.tar.gz"
```

Reply:
dl.google.com also works in some countries like China Mainland.